.husky/commit-msg

```diff
@@ -3,7 +3,7 @@ COMMIT_MSG=$(cat "$COMMIT_MSG_FILE")
 
 COMMIT_REGEX="^(MRJ-\d+ - .+|Merge .*)"
 
-if ! echo "$COMMIT_MSG" | grep -Eq "$COMMIT_REGEX"; then
+if ! echo "$COMMIT_MSG" | grep -Pq "$COMMIT_REGEX"; then
```

Comment:
not sure this a valid change, why do you need it?

Reply:
as i mentioned, i was having an issue with the regex pattern. \d was not recognized by grep -E, so i changed it to grep -P.